dashboard/app/views/blocks/_load_blockly.html.haml

```diff
@@ -0,0 +1,7 @@
+- if @pool == 'Dancelab' || @pool == 'Poetry'
+  %script{src: webpack_asset_path('js/googleblockly.js')}
+  - @blockly_version = 'google'
+- else
+  %script{src: webpack_asset_path('js/blockly.js')}
+  %script{src: webpack_asset_path("js/#{js_locale}/blockly_locale.js")}
```

Comment:
@molly-moen - would you mind explaining why line 6 isn't need for google blockly version? @mikeharv was asking about this in my [current PR - still in draft mode](https://github.com/code-dot-org/code-dot-org/pull/52034/files#r1202915445).

Reply:
I don't remember 😬